h2/src/main/org/h2/command/dml/RunScriptCommand.java

```diff
@@ -68,7 +69,20 @@ public long update() {
                 if (sql == null) {
                     break;
                 }
-                execute(sql);
+
+                // Normalize the statement
+                String upper = StringUtils.toUpperAscii(sql);
```

Comment:
That's just a waste of CPU and memory resources, `String.equalsIgnoreCase()` is expected to be much cheaper.